cmd/prometheus/config.go

```diff
@@ -190,45 +187,6 @@ func init() {
 		"Local storage engine. Supported values are: 'persisted' (full local storage with on-disk persistence) and 'none' (no local storage).",
 	)
 
-	// Remote storage.
-	cfg.fs.StringVar(
```

Comment:
> Isn't it more dangerous to keep the flags and silently break things for users rather than obviously break it? 

That's my viewpoint, however the doc changed under me and that's not what we ended up agreeing.